### ISSUE TITLE
font-awesome inside b-form-group labels.

### ISSUE THREAD

**Issue by ghenry** [bootstrap-vue/bootstrap-vue#2070]
Hi,
Has anyone been able to do:

```
  <b-form-group label='<i class="fas fa-envelope fa-fw'/> Enter your name">
  </b-form-group>
```

etc?

**Comment by lggwettmann** [comment]
Haven't tried that yet. I'm using vue-font-awesome to display font awesome icons. Perhaps you can come up with a way of creating an own b-form-group variation with that?

**Comment by ghenry** [comment]
I'm using this https://dobromir-hristov.github.io/vuelidate-error-extractor/other_frameworks.html#usage-with-bootstrapvue and have added a prop to allow me to pass in a FA class. How are you getting on with vue-font-awesome? I opted against it as it didn't bring me much over CSS. What do you like about it? (not the right place for this I know)

**Comment by lianee** [comment]
does this work for you?
`<b-form-group>
<slot name="label"><i class="fas fa-envelope fa-fw"></i> Enter your name</slot>
</b-form-group>`

**Comment by ghenry** [comment]
No, the class of fa is used and it's all in one small space. Did try that.

**Comment by lianee** [comment]
is that not what you want?
[https://jsfiddle.net/6tq953cw/](https://jsfiddle.net/6tq953cw/)

if not, perhaps a picture would help me to understand

**Comment by zoladkow** [comment]
and in case someone would want to us FA inside input groups, this works quite well:
```html
<b-input-group>
    <b-input-group-prepend>
        <div class="input-group-text"><i class="fas fa-user"></i></div>
    </b-input-group-prepend>
    <b-form-input v-model="username" type="text" id="login-username"
                  placeholder="Enter your name"></b-form-input>
</b-input-group>
```
